midi_app_controller/gui/utils.py

```diff
@@ -141,9 +141,9 @@ def get_selected_action_id(self) -> Optional[str]:
 
 def is_subpath(path: Path, subpath: Path) -> bool:
     """Checks if one path represents a file/directory inside the other directory."""
-    path_str = str(subpath.resolve().absolute())
-    subpath_str = str(path.resolve().absolute())
-    return path_str.startswith(subpath_str)
+    path_str = str(path.resolve().absolute())
+    subpath_str = str(subpath.resolve().absolute())
+    return subpath_str.startswith(path_str)
```

Comment:
Regarding comment by @lukaszsmolinski, I think this version is correct and when I first wrote the function I meant it like this, with `path = A/B, subpath = A/B/C -> True`, so it is called everywhere correctly. I'm not sure why the variables were flipped in the function body. 

However, it has come to my attention "subpath" is ambiguous and can mean opposite things in this context. I'd change the docstring to ``"Checks if `subpath` represents a file/directory inside the directory represented by `path`"``.

Reply:
Yes, now it works. When I reviewed it, the last line was `path_str.startswith(subpath_str)`.